com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java

```diff
@@ -409,15 +411,17 @@ private String getDBLabel() {
 	private Composite createAutoLayoutComp() {
 		Composite comp = new Composite(this, SWT.NONE);
 		final GridLayout gdLayout = new GridLayout(1, false);
-		gdLayout.marginHeight = 1;
-		gdLayout.marginWidth = 1;
-		gdLayout.horizontalSpacing = 2;
-		gdLayout.verticalSpacing = 0;
+		final GridData gridData = new GridData(SWT.FILL, SWT.FILL, true, true);
+		
 		comp.setLayout(gdLayout);
-		comp.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
-
-		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER);
-		autoLayoutButton.setLayoutData(CommonUITool.createGridData(1, 1, 98, 20));
+		comp.setLayoutData(gridData);
+
+		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER | SWT.FILL);
+		Font font = autoLayoutButton.getFont();
+		String fontName = font.getFontData()[0].name;
+		int fontSize = (int)font.getFontData()[0].height;
```

Comment:
`FontData.height` is same as `FontData.name` too.
We should change like below.
`int fontSize = font.getFontData()[0].getHeight();`